R_scripts/licorplot.R

```diff
@@ -13,41 +13,51 @@ library(readr)
 source("licordat.R")
 licorDat <- read_dir("../licor_data/")
 collarDat <- read_csv("../design/cores_collars.csv")
+plots <- read_csv("../design/plots.csv")
```

Comment:
Is there a huge difference between _read_csv_ and _read.csv_?

Reply:
`read.csv` is the R default, and produces a data frame. It's robust but not fast.

`read_csv` is provided by the `readr` package. It's a little more fragile in some ways, but much faster, produces a tibble, and automatically handles things like dates and times.